roachtest: deflake splits/largerange/size=32GiB

iskettaneh opened the pull request:
This commit reduces the snapshot_rebalance.max_rate from 512 to 192. The reasons are: (1) Even on Azure and GCE, I saw that we get about 192 Mbps of an actual snapshot send rate. (2) On IBM, the rate is much lower. (3) This rate goes into calculating the snapshot sending timeout, and it should generally be lower or slightly higher than the actual snapshot sending rate. If it's 10 times lower than the actual rate, the snapshot will most likely timeout.

See this comment for the exact analysis:
https://github.com/cockroachdb/cockroach/issues/148982#issuecomment-3020552145

Fixes: #148982

Release note: None

tbg reviewed (dismissed):
I think you accidentally checked in a pprof file :)
LGTM otherwise.

iskettaneh committed:
roachtest: deflake splits/largerange/size=32GiB

This commit reduces the snapshot_rebalance.max_rate from 512 to 192. The
reasons are: (1) Even on Azure and GCE, I saw that we get about 192 Mbps
of an actual snapshot send rate. (2) On IBM, the rate is much lower.
(3) This rate goes into calculating the snapshot sending timeout, and
it should generally be lower or slightly higher than the actual
snapshot sending rate. If it's 10 times lower than the actual rate, the
snapshot will most likely timeout.

See this comment for the exact analysis:
https://github.com/cockroachdb/cockroach/issues/148982#issuecomment-3020552145

Fixes: #148982

iskettaneh commented:
TFTR!

bors r+
